Software_Carbon_Intensity/Appendix_A.md

```diff
@@ -1 +1,101 @@
+
+Thus we can consider that the metric we will use for the site reliability engineer will be the Total carbon emissions (C) value. The formula for this metric is :C= O+ M where O= E*I
+
+**Scope of Metric**
+For this metric, the definition of scope around which SCI will operate is very important. Scope is the boundary area where we will apply the monitoring. Since we are talking about software, the boundary here is the software boundary as defined in the SCI specification. 
+However, we may not be able to apply this uniformly for all software. Software varies by architecture , environment, hosting type (dedicated infrastructure vs shared infrastructure vs serverless ) and the implementation of SRE monitoring for SCI varies by these factors .We will discuss these factors below:
+
+1) Architecture of Software
+Different application architectures need monitoring differently. Consider for example the following workloads:
+
+	1) Web based multi-tier application or Long running process deployed on either cloud or on-premise
+	2) Mobile app connecting to backend APIs on cloud or on-premise
+	3) Desktop app connecting to backend APIs on cloud or on-premise
+	4) AI based machine learning model experiments
+	5) Open Source or Closed Source Framework SDKs
+	6) Server less applications
+
+From a pure monitoring perspective of the SRE metric , doing it on server based workloads in the above list may be the first step . For e.g Web based multi-tier applications have either Virtual machines or EC2s connecting to APIs and databases and hence we can monitor the Operational emissions of these server components.  Similarly we can calculate the metric for backend server APIs and serverless components of the Mobile and desktop apps.
+
+There would be challenges however in doing the same for desktop devices and mobile platforms as the emissions calculations would need to know at a rough level the total number of  mobile devices or desktops , their types etc to calculate roughly the value for the desktop apps.   Hence for the first release of specifications, we can propose that monitoring will be for a subset of the above workloads i.e for workloads which have majorly server components.
+
+
+2) Hosting Infrastructure - Dedicated Vs Shared
+
+Monitoring techniques will also vary based on the hosting mode for the software. For those with dedicated infrastructure, SCI will just be the sum total of Operational Emission values across the different layers. In the equation for SCI ( SCI per unit of baseline = (E*I) + M , the value of M does not make an impact when calculating delta carbon intensity = Current CI- Original CI since the hardware is exclusively reserved for the purpose of the said software. Hence the monitoring technique can potentially look for variances in the Operational Emissions value and variances of it to raise alert for the Site Reliability Engineer.
```

Comment:
```suggestion
Monitoring techniques will also vary based on the hosting mode for the software. For those with dedicated infrastructure, the SCI will just be the sum total of Operational Emission values across the different layers. In the equation for SCI ( SCI per unit of baseline = (E*I) + M , the value of M does not make an impact when calculating delta carbon intensity = Current CI- Original CI since the hardware is exclusively reserved for the purpose of the said software. Hence, the monitoring technique can potentially look for variations in the Operational Emissions value and variations of it to raise alerts for the SRE stakeholder.
```

Reply:
@srini1978 in the following sentence: 

>In the equation for SCI ( SCI per unit of baseline = (E*I) + M , the value of M does not make an impact when calculating delta carbon intensity = Current CI- Original CI since the hardware is exclusively reserved for the purpose of the said software.

I think you are missing the closing parenthesis. Additionally, it is a compound sentence that makes it hard to read. I would suggest breaking it up to make it easier to understand. Thanks!